frontend/test/metabase/lib/formatting.unit.spec.js

```diff
@@ -415,7 +415,7 @@ describe("formatting", () => {
           time_style: "HH:mm",
           column: {
             base_type: "type/Date",
-            unit: "hour-of-day",
+            unit: "minute",
```

Comment:
Why are we changing specs? Did the behavior change?

Reply:
In the original formatting code, the unit is ignored and the `date-style` is passed directly to Moment. The new code sees a time-only unit of `hour-of-day` and disables the date output. Then there's no time nor date output and we get a default ISO string `2019-07-07T00:00:00Z` out.

This seemed like a test bug, so I adjusted the input. The test is actually there to test that for a column of `type/Date` the time part is suppressed even when the other options say it's enabled. Changing to a time-allowed unit of `minutes` matches that.